src/App.js

```diff
@@ -1,16 +1,95 @@
 import React from 'react';
+// import Todo from './components/TodoComponents/Todo';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+
+const todoArr = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
+
+
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      todoArrObject: todoArr,
+      task: "",
+      id: "",
+      completed: ""
+    };
```

Comment:
So you don't really want to have these properties on state outside of your todoArrObject:

``` 
task: "",
      id: "",
      completed: ""
```

Right? These are the properties of your individual todo object. So they should exist inside the objects inside of your todoArr.